exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return (number < 0);
```

Comment:
Nice work Nishka.
Since a predicate function returns a boolean, you can very often simply do as Nishka did in this function and return the result of a  boolean expression without the need for a IF statement.

The function could be written like this as well, but as we'll see later, its often simpler to drop the IF statement and simply return the result of some expression :
```suggestion
  if (number < 0) {   // this results or "evaluates" to either boolean value true or false
     return true;
     } else {
        return false;
       }
```